cloud_governance/policy/aws/cost_explorer.py

```diff
@@ -11,6 +11,10 @@
 
 
 class CostExplorer(ElasticUpload):
+    """
+    Fetch the cost_explorer reports from the aws based on the daily granularity,
+    and upload to ElasticSearch then create dashboards.
+    """
```

Comment:
This class fetches the cost_explorer reports from the AWS based on the daily and upload to ElasticSearch.